src/utils/handleTicketDelete.ts

```diff
@@ -150,3 +151,28 @@ export const handleTicketDelete = async (
 		console.error(err);
 	}
 };
+
+const fetchMessages = async (
+	channel: TextChannel | ThreadChannel,
+	messages: Message[],
```

Comment:
```suggestion
	messages: Message[] = [],
```
Set the default value as an empty array, so it doesn't have to be specified